backend/ibutsu_server/scripts/mongo2postgres.py

```diff
@@ -316,17 +327,21 @@ def migrate_tables(mongo, vprint, migrate_files=False):
         # migrate the table over
         vprint("Migrating {} ".format(collection), end="")
         if collection == "runs":
-            run_ids = migrate_table(mongo[collection], model, vprint, filter_=filter_)
+            run_ids = migrate_table(
+                mongo[collection], model, vprint, filter_=table_filter, limit=record_limit
+            )
         elif collection == "results":
             # migrate in chunks of 100 runs at a time
             run_chunks = [run_ids[i : i + 100] for i in range(0, len(run_ids), 100)]
             for run_list in run_chunks:
                 result_filter = {"metadata.run": {"$in": run_list}}  # filter on runs we know exist
                 result_ids.extend(
-                    migrate_table(mongo[collection], model, vprint, filter_=result_filter)
+                    migrate_table(
+                        mongo[collection], model, vprint, limit=record_limit, filter_=result_filter
```

Comment:
Because the results are migrated in chunks of 100 runs at a time, the record_limit will probably never be applied here.